apps/builder/app/canvas/features/text-editor/text-editor.tsx

```diff
@@ -1022,10 +1046,10 @@ const ContextMenuPluginInternal = ({
           return false;
         }
 
-        if (!isSingleCursorSelection()) {
-          closeMenu();
-          return false;
-        }
+        //if (!isSingleCursorSelection()) {
```

Comment:
not needed?

Reply:
Probably needed will leave a little bit here. Not sure about bugs